### PR TITLE
add november 2023 draft newsletter

### PR DESCRIPTION
**Pull request by MILeach**

### CODE REVIEW

File: _newsletters/2023-11-02-newsletter.md [hunk 1]
```diff
@@ -0,0 +1,185 @@
...
+### Digital Research Practice Support Community
+
+The DRPS community is a group for people that support researchers in carrying out research in the digital age. Meetings
+are held monthly, with discussions around events, training and opportunities related to the field.
+
+You can join the google group
+[here](https://groups.google.com/u/1/a/sheffield.ac.uk/g/digital-research-practice-support-community-group/about) to
+stay informed.
+
+The next meeting is scheduled for 2pm on Wednesday 9th August.
```
Comment (davidwyld) [inline _newsletters/2023-11-02-newsletter.md]:
```suggestion
The next meeting is scheduled for 2pm on Wednesday 8th November.
```

File: _newsletters/2023-11-02-newsletter.md [hunk 2]
```diff
@@ -0,0 +1,185 @@
...
+- You can now apply for the [Open Science Retreat](https://openscienceretreat.eu/) (25th - 29th March, 2024). This event brings people together to share their experiences of and best practices for open science.
+- The Software Sustainability Institute (SSI) have announced their next collaborations workshop, to be held 30th April - 2nd May 2024. You can now [register your interest](https://software.us7.list-manage.com/subscribe?u=628ee3fc74b8ef3497e3a7b66&id=9616c70676)
+- The N8 CIR are hosting a [half-day event](https://n8cir.org.uk/events/data-access-for-digital-health/) on the 8th November to discuss the challenges of data access in the field of digital health and what organisations and researchers are doing to overcome this challenge.
+- The N8 CIR are hosting [an event on Synthetic Data Generation](https://n8cir.org.uk/events/synthetic-data-generation-event/) on the 28th November.
+- NVIDIA are hosting a free [AI and Data Science Virtual Summit](https://www.nvidia.com/en-us/events/ai-data-science-virtual-summit/).
+- The [PyLadies Conference](https://conference.pyladies.com/) will be running from 30th November to 3rd December 2023.
+
+### Training
```
Comment (davidwyld) [inline _newsletters/2023-11-02-newsletter.md:61]: I would take out the whole training section here, there aren't any events organised by the RSE team scheduled and the CRS site is now defunct.

**Review by davidwyld — CHANGES_REQUESTED** [review]
Looking good! Thanks @MILeach - one or two suggestions.

File: _newsletters/2023-11-02-newsletter.md [hunk 3]
```diff
@@ -0,0 +1,185 @@
...
+
+#### External Training
+
++ [The Turing Online Learning Platform](https://www.turing.ac.uk/courses) contains a number of courses, many of which
+  can be undertaken independently.
++ [4th OpenAIRE Open Science Train-the-Trainer
+  Bootcamp](https://www.openaire.eu/4th-openaire-open-science-train-the-trainer-bootcamp)
+- Durham University are hosting a [workshop series on debugging, testing and correctness](https://scicomp.webspace.durham.ac.uk/teaching/professional-development/debugging-testing-and-correctness-workshop-series-2023/) for massively parallel codes
+<!--#### Research IT Training-->
+
```
Comment (ns-rse) [inline _newsletters/2023-11-02-newsletter.md:90]: Moving this from above to where it should be sorry.

Its all wrapped up and hidden under MyDevelopment which I don't find particularly easy to navigate. There is no useful hierarchical structure and I only found them by searching for nothing but filtering Vendor for "IT Services".

Each course has at least one event scheduled, some up to 5. Some that are due to run soon have 65 places available, others are fully booked.

No idea where these have been advertised :shrug: , I don't recall seeing it on the RSE mailing list.


## R

* [R for Statistics and Data Visualisation](https://mydevelopment.csod.com/ui/lms-learning-details/app/event/943cfaf5-cb85-4b58-9af7-16064ae8752b) 2023-11-09 (fully booked, 5 people on waiting list)

## Python

* [Python for Data Science I](https://mydevelopment.csod.com/ui/lms-learning-details/app/event/653bbdfb-ab61-451b-894f-c38de64f7dbc) 2023-21-05 (fully booked, 5 people on waiting list)
* [Python Programming 1](https://mydevelopment.csod.com/ui/lms-learning-details/app/event/d2c02e5b-4b2e-43dd-9860-07ccca5c0400) 2023-11-27 (39 places available) 2023-12-11 (fully booked, 2 people on waiting list).
* [Python Programming 2](https://mydevelopment.csod.com/ui/lms-learning-details/app/event/8ebd2c6f-2c1b-499f-9866-9c78ae625172) 2023-12-01 (65 places available) 2023-12-12 (3 places available).
* [Python Programming 3](https://mydevelopment.csod.com/ui/lms-learning-details/app/event/d54e003f-6948-4e89-b245-57e4a1b5b587) 2023-12-04 (65 places available) 2024-01-16 (2 places available).
* [Supervised Machine Learning 1](https://mydevelopment.csod.com/ui/lms-learning-details/app/event/184a44dc-acdd-4908-8246-3d9897492f06) 2023-11-17 (14 places available) 2023-11-30 (91 spaces available) 2023-12-14 (7 spaces available) 2024-01-12 (13 spaces available) 2024-02-02 (13 spaces available)
* [Supervised Machine Learning 2](https://mydevelopment.csod.com/ui/lms-learning-details/app/event/87c9ec1e-648a-4d60-8a99-0a7eccf53577) 2024-01-15 (fully booked, 2 people on waiting list) 2024-01-29 (10 spaces available)
*  [Temporal Analysis in Python](https://mydevelopment.csod.com/ui/lms-learning-details/app/event/31ad16b9-5d28-48cf-8a46-1437e2248c8c) 2023-11-24 (4 places available) 2024-01-19 (6 spaces available)

Its quite bizzare that the advanced courses have more sessions scheduled than the intro ones, perhaps some have already run :thinking: 

## Matlab

* [Introduction to Matlab 1](https://mydevelopment.csod.com/ui/lms-learning-details/app/event/d96972b1-cb9e-475e-b680-a83d3fa57972) 2023-11-09 (1 place available)
* [Introduction to Matlab 2](https://mydevelopment.csod.com/ui/lms-learning-details/app/event/0746d395-7a7d-45b9-9cee-3f7220e28202) 2023-11-30 (9 places available)

File: _newsletters/2023-11-02-newsletter.md [hunk 4]
```diff
@@ -0,0 +1,185 @@
...
+### Opportunities
+
+#### Jobs
+
+You can always check for advertised RSE and RSE-adjacent roles over at the [RSE society's vacancies
+board](https://society-rse.org/careers/vacancies/).
+
+- Grade 9 [Principal Research Infrastructure](https://www.ucl.ac.uk/work-at-ucl/search-ucl-jobs/details?nPostingId=6263&nPostingTargetId=16767&id=Q1KFK026203F3VBQBLO8M8M07&LG=UK&languageSelect=UK&mask=extHi) Developer at UCL. Applications close 23rd November.
+- The University of Sheffield are offering a [PhD in the School of Biosciences](https://www.findaphd.com/phds/project/bis-01-mesnage-mass-xplorer-a-transformative-web-based-software-platform-for-peptidoglycomics-a-new-discipline-to-explore-antimicrobial-resistance-and-bacterial-population-dynamics/?p163266), with the aim of developing a web-based software platform for peptidoglycomics, a new discipline to explore antimicrobial resistance and bacterial population dynamics.
+#### Volunteer Positions
```
Comment (ns-rse) [inline _newsletters/2023-11-02-newsletter.md:111]: Pedantic Markdown linting (see pending #700 ) would want a space between paragraph and new lines (would also wrap lines at 120 characters).

File: _newsletters/2023-11-02-newsletter.md [hunk 5]
```diff
@@ -0,0 +1,185 @@
...
+- rOpenSci, who promote open and reproducible research have published their [latest newsletter](https://ropensci.org/blog/2023/10/20/news-october-2023/).
+
+#### MATLAB
+- MATLAB R2023b includes updated support for [gate-based quantum computing](https://blogs.mathworks.com/matlab/2023/10/17/quantum-computing-in-matlab-r2023b-on-the-desktop-and-in-the-cloud/)
+
+
+#### Open Research
+- The University of Sheffield will be hosting its [Annual Open Research Lecture](https://www.sheffield.ac.uk/openresearch/events/annual-open-research-lecture) on the 6th December 2023.
+- The University of Sheffield is hosting a [series of bitesize seminars](https://www.sheffield.ac.uk/openresearch/events/how-share-seminar-series) on how best to share different types of research data.
+- Free Software Foundation Europe (FSFE) published a [short explainer video on free software](https://media.fsfe.org/w/p/9gYSyoEYggsqBExLWjRejL).
```
Comment (Robadob) [inline _newsletters/2023-11-02-newsletter.md:35]: is this in the right section?

File: _newsletters/2023-11-02-newsletter.md [hunk 6]
```diff
@@ -0,0 +1,185 @@
...
+#### Blogs
+- Eric Jensen and Daniel Katz write on [What we learned about research software careers from a recent international workshop](https://urssi.us/blog/2023/10/24/what-we-learned-about-research-software-careers-from-a-recent-international-workshop/).
+- Julia Evans presents a talk titled [Making Hard Things Easy](https://jvns.ca/blog/2023/10/06/new-talk--making-hard-things-easy/), exploring how we can better tackle learning difficult to grasp technologies.
+
+
+#### Publications
+
+
+### Events
+- The Alan Turing Institute is hosting the UK’s national showcase of data science and artificial intelligence (AI), [AI UK 2024](https://ai-uk.turing.ac.uk/). Tickets are now available.
```
Comment (Robadob) [inline _newsletters/2023-11-02-newsletter.md]: 19-24 march

File: _newsletters/2023-11-02-newsletter.md [hunk 7]
```diff
@@ -0,0 +1,185 @@
...
+- You can now apply for the [Open Science Retreat](https://openscienceretreat.eu/) (25th - 29th March, 2024). This event brings people together to share their experiences of and best practices for open science.
+- The Software Sustainability Institute (SSI) have announced their next collaborations workshop, to be held 30th April - 2nd May 2024. You can now [register your interest](https://software.us7.list-manage.com/subscribe?u=628ee3fc74b8ef3497e3a7b66&id=9616c70676)
+- The N8 CIR are hosting a [half-day event](https://n8cir.org.uk/events/data-access-for-digital-health/) on the 8th November to discuss the challenges of data access in the field of digital health and what organisations and researchers are doing to overcome this challenge.
```
Comment (Robadob) [inline _newsletters/2023-11-02-newsletter.md]: 2023!
Be useful to make this list chronological.

File: _newsletters/2023-11-02-newsletter.md [hunk 8]
```diff
@@ -0,0 +1,185 @@
...
+#### Jobs
+
+You can always check for advertised RSE and RSE-adjacent roles over at the [RSE society's vacancies
+board](https://society-rse.org/careers/vacancies/).
+
+- Grade 9 [Principal Research Infrastructure](https://www.ucl.ac.uk/work-at-ucl/search-ucl-jobs/details?nPostingId=6263&nPostingTargetId=16767&id=Q1KFK026203F3VBQBLO8M8M07&LG=UK&languageSelect=UK&mask=extHi) Developer at UCL. Applications close 23rd November.
+- The University of Sheffield are offering a [PhD in the School of Biosciences](https://www.findaphd.com/phds/project/bis-01-mesnage-mass-xplorer-a-transformative-web-based-software-platform-for-peptidoglycomics-a-new-discipline-to-explore-antimicrobial-resistance-and-bacterial-population-dynamics/?p163266), with the aim of developing a web-based software platform for peptidoglycomics, a new discipline to explore antimicrobial resistance and bacterial population dynamics.
+#### Volunteer Positions
+
+- The [UK RSECon](https://society-rse.org/rse-conference-2024/) and [US RSECon](https://docs.google.com/forms/d/e/1FAIpQLSfxkbo-6eF2NENsrWwb44_Bb32UNCzgP2d6-z3Hq8HxbYCWYA/viewform) are each looking for volunteers for their organising committees.
```
Comment (Robadob) [inline _newsletters/2023-11-02-newsletter.md]: Worth highlighting the deadline, I think RSECON UK deadline is Tuesday next week. So VERY close.

File: _newsletters/2023-11-02-newsletter.md [hunk 9]
```diff
@@ -0,0 +1,185 @@
...
+- The University of Sheffield are offering a [PhD in the School of Biosciences](https://www.findaphd.com/phds/project/bis-01-mesnage-mass-xplorer-a-transformative-web-based-software-platform-for-peptidoglycomics-a-new-discipline-to-explore-antimicrobial-resistance-and-bacterial-population-dynamics/?p163266), with the aim of developing a web-based software platform for peptidoglycomics, a new discipline to explore antimicrobial resistance and bacterial population dynamics.
+#### Volunteer Positions
+
+- The [UK RSECon](https://society-rse.org/rse-conference-2024/) and [US RSECon](https://docs.google.com/forms/d/e/1FAIpQLSfxkbo-6eF2NENsrWwb44_Bb32UNCzgP2d6-z3Hq8HxbYCWYA/viewform) are each looking for volunteers for their organising committees.  
+- The Journal of Open Source Software (JOSS) has issued a [call for editors](https://blog.joss.theoj.org/2023/10/call-for-editors).
+- The Carpentries, who promote inclusive teaching of data and coding skills, are [looking for candidates for their board of directors](https://carpentries.org/blog/2023/09/board-of-directors-election-announcement/).
```
Comment (Robadob) [inline _newsletters/2023-11-02-newsletter.md]: Deadline was late October, this is defunct.

File: _newsletters/2023-11-02-newsletter.md [hunk 10]
```diff
@@ -0,0 +1,185 @@
...
+
+- Grade 9 [Principal Research Infrastructure](https://www.ucl.ac.uk/work-at-ucl/search-ucl-jobs/details?nPostingId=6263&nPostingTargetId=16767&id=Q1KFK026203F3VBQBLO8M8M07&LG=UK&languageSelect=UK&mask=extHi) Developer at UCL. Applications close 23rd November.
+- The University of Sheffield are offering a [PhD in the School of Biosciences](https://www.findaphd.com/phds/project/bis-01-mesnage-mass-xplorer-a-transformative-web-based-software-platform-for-peptidoglycomics-a-new-discipline-to-explore-antimicrobial-resistance-and-bacterial-population-dynamics/?p163266), with the aim of developing a web-based software platform for peptidoglycomics, a new discipline to explore antimicrobial resistance and bacterial population dynamics.
+#### Volunteer Positions
+
+- The [UK RSECon](https://society-rse.org/rse-conference-2024/) and [US RSECon](https://docs.google.com/forms/d/e/1FAIpQLSfxkbo-6eF2NENsrWwb44_Bb32UNCzgP2d6-z3Hq8HxbYCWYA/viewform) are each looking for volunteers for their organising committees.  
+- The Journal of Open Source Software (JOSS) has issued a [call for editors](https://blog.joss.theoj.org/2023/10/call-for-editors).
```
Comment (Robadob) [inline _newsletters/2023-11-02-newsletter.md]: Deadline 6th Nov, very close!

File: _newsletters/2023-11-02-newsletter.md [hunk 11]
```diff
@@ -0,0 +1,185 @@
...
+- The Carpentries, who promote inclusive teaching of data and coding skills, are [looking for candidates for their board of directors](https://carpentries.org/blog/2023/09/board-of-directors-election-announcement/).
+
+#### Funding
+- The University of Sheffield's [Unleash your Data and Software competition](https://www.sheffield.ac.uk/library/research-data-management/unleash-your-data-2023-24) has launched! PGRs and staff can apply for an award of up to £5,000 for a project. The deadline is 1st December, with two drop-in sessions for people to talk through their potential applications on 15th November, and 21st November
+- The Open Bioinformatics Foundation have opened a call for applications for the [OBF Event Fellowship](https://www.open-bio.org/2023/10/19/event-fellowship-2023-3/) which aims to increase diversity in attendance of events promoting open-source bionformatics or open science.
```
Comment (Robadob) [inline _newsletters/2023-11-02-newsletter.md]: Deadline 1st Dec

File: _newsletters/2023-11-02-newsletter.md [hunk 12]
```diff
@@ -0,0 +1,185 @@
...
+stay informed.
+
+The next meeting is scheduled for 2pm on Wednesday 9th August.
+
+### LunchBytes
+
+[LunchBytes](https://rse.shef.ac.uk/community/lunch-bytes/) are short talks from the research community on research
+software, data, and infrastructure.
+
++ [Making research software more visible and reusable: Open Source Software and
```
Comment (Robadob) [inline _newsletters/2023-11-02-newsletter.md:136]: 6th Dec 12:00-13:00

**Review by Robadob — CHANGES_REQUESTED** [review]
See my individual comments, mostly about highlighting deadlines.

**Review by Robadob — APPROVED** [review]
Looks good.

Little unsure about the "full, n people on waiting list" level of detail, but eh.

**Pull request merged**